lib/taste_tester/config.rb

```diff
@@ -64,6 +65,7 @@ module Config
     json false
     jumps nil
     windows_target false
+    parallel_hosts Etc.nprocessors
```

Comment:
This is where I default it. The number of cores should be safe for most systems, but happy to put something smaller.